cmd/gdeploy/commands/sso/sso.go

```diff
@@ -176,12 +179,45 @@ var configureCommand = cli.Command{
 Create a group called 'Granted Administrators' in your identity provider and copy the group's ID.
 Users in this group will be able to manage Access Rules.
 `)
-		adminGroupPrompt := &survey.Input{Message: "The ID of the Granted Administrators group in your identity provider:", Default: dc.Deployment.Parameters.AdministratorGroupID}
-		err = survey.AskOne(adminGroupPrompt, &dc.Deployment.Parameters.AdministratorGroupID)
+
+		err = idp.IdentityProvider.Init(ctx)
+		if err != nil {
+			return err
+		}
+
+		err = deploy.RunConfigTest(ctx, idp.IdentityProvider)
+		if err != nil {
+			return err
+		}
+		grps, err := idp.IdentityProvider.ListGroups(ctx)
 		if err != nil {
 			return err
 		}
 
+		// convert groups to a string map
+		groupMap := make(map[string]identity.IdpGroup)
+		groupNames := []string{}
+		chosenKey := ""
+		for _, g := range grps {
+			key := fmt.Sprintf("%s: %s", g.Name, g.Description)
+			groupMap[key] = g
+			groupNames = append(groupNames, key)
+		}
+
+		// sort groupNames alphabetically
+		sort.Strings(groupNames)
+
+		err = survey.AskOne(&survey.Select{
+			Message: "The ID of the Granted Administrators group in your identity provider:",
+			Options: groupNames,
+		}, &chosenKey)
+
+		if err != nil {
+			return err
+		}
+
+		dc.Deployment.Parameters.AdministratorGroupID = groupMap[chosenKey].Name
```

Comment:
This needs to be the ID rather than the name
```suggestion
		dc.Deployment.Parameters.AdministratorGroupID = groupMap[chosenKey].ID
```